src/ChangelogGenerator.php

```diff
@@ -117,14 +118,12 @@ private function collectReleaseIssues($startDate = null)
          do {
              $currentRelease = current($releases);
 
-            if ($startDate &&
-                date_diff(new DateTime($currentRelease->published_at), new DateTime($startDate))->days <= 0
-            ) {
+            if ($startDate && date_diff(new DateTime($currentRelease->published_at), $startDate)->days <= 0) {
```

Comment:
i'm fine with this `startDate` change, when this pr gets merged a new major version will be released to indicate bc breaks

Reply:
This doesn't change the functionality of the public API, so it keeps within semver specs. I would suggest keeping this package in the major version of 0 for a few more releases, though. Semver describes a major version of 0 to be used for "...initial development. Anything may change at any time. The public API should not be considered stable." what are your thoughts?

Reply:
You are totaly right!